.github/workflows/wheels.yml

```diff
@@ -2,8 +2,8 @@
 name: Wheel Builds
 on:
   push:
-    tags:
-      - '*'
+    pull_request:
+      branches: ['main']
```

Comment:
I think the trick is that `pull_request` isn't part of `push`, it's
```yaml
on:
  pull_request:
    ...
```

Reply:
ah good catch. Well if I need to rerun https://github.com/Qiskit/qiskit-terra/actions/runs/2071342031 for some reason I'll fix this here and delete that branch. But hopefully everything will just work